ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -89,7 +89,7 @@ export class Auth0AdapterService implements OnDestroy {
                 redirectUri
             });
         } catch (e) {
-            this.log.logError(`${this.LOG_SOURCE}.createWebAuth`, e);
+            this.log.logToCloud(`${this.LOG_SOURCE}.createWebAuth`, e);
```

Comment:
Alex, the function logToCloud() is specifically employed to transmit general information, while logError() is designed for recording and tracking errors. If we were to exclusively rely on logToCloud(), any errors occurring wouldn't be captured and registered within the cloud-based logs

P.S. If you aim to enhance the comprehensiveness of your logs, rather than using plain strings, you can simply provide an object as an argument to the logError method. This allows you to define and include all the pertinent information

Reply:
+ logToCloud is observable. It won't work if you didn't subscribe